src/instruments/buildSrc/directories.mjs

```diff
@@ -1,11 +1,13 @@
 import os from 'os';
+import path from 'path';
```

Comment:
These imports should be consistent, we use `import { join } from 'path'` in all other files.